phalcon/config.zep

```diff
@@ -79,6 +81,61 @@ class Config implements \ArrayAccess, \Countable
 	}
 
 	/**
+	 * Sets the default path delimiter
+	 */
+	public function setPathDelimiter(string! delimiter) -> <Config>
+	{
+		let self::_delimiter = delimiter;
+		return this;
+	}
+
+	/**
+	 * Returns a value from current config using a dot separated path.
+	 *
+	 *<code>
+	 * echo $config->get("unknown.path", "default", ".");
+	 *</code>
+	 */
+	public function path(var path, var defaultValue = null, var delimiter = null) -> var
+	{
+		var key, keys, config, temp;
+
+		let path = strval(path);
+
+		if isset this->{path} {
+			return this->{path};
+		}
+
+		if empty delimiter {
+			let delimiter = self::_delimiter;
+		}
+
+		let keys = explode(delimiter, path),
+			config = this;
+
+		while !empty keys {
+			let key = array_shift(keys);
+
+			if !isset config->{key} {
+				break;
+			}
+
+			if empty keys {
+				return config->{key};
+			}
+
+			let temp = config->{key};
+			let config = temp;
```

Comment:
I thought I found the PHP7 test error. Previously I wrote 
let config = config->{key}